src/server/cea_608_encoder/caption_string_utility.py

```diff
@@ -141,20 +144,44 @@ def create_byte_pairs_for_caption_string(caption_string: str) -> list:
     byte_pairs = bytes_to_byte_pairs(raw_hex_values)
     return byte_pairs
 
+def create_byte_pairs_for_midrow_style(color: str, underline = False):
+    byte_list = []
+    
+    # Default: text color: white, no underline
+    first_byte = 0x11 
+    second_byte = 0x20
+
+    if isinstance(color, str):
+        color = color.lower()
+    if color in text_colors:
+        second_byte += text_colors[color]
+    if underline == True: 
+        second_byte += 0x1  
+
+    bytes = []
+    bytes.append(first_byte)
+    bytes.append(second_byte)
+
+    # To move the cursor one position back
+    # CC1 Backspace control command
+    bytes.append(0x14)
+    bytes.append(0x21)
+
+    for byte in bytes:
+        if check_parity(byte) == 0:
+            byte = add_parity_to_byte(byte)
+        byte_list.append(hex(byte))
 
-def create_byte_pairs_for_caption_color(color):
-    pass
-
-
-def create_byte_pairs_for_text_alignment(alignment):
-    pass
+    raw_hex_values = parse_raw_hex_values(byte_list)
+    byte_pairs = bytes_to_byte_pairs(raw_hex_values)
+    return byte_pairs
 
 
-def create_bytes_to_underline_text():
+def create_byte_pairs_for_caption_color(color):
```

Comment:
this function and the one below are being used in other parts of he code, changing the name here needs to change the dependencies in other file too. or we will run into the same problem we had to fix before

Reply:
I did not rename any functions. I don't know why github displays it this way. You can compare it with the develop branch and "create_byte_pairs_for_caption_color" and the other one are already in the develop branch. I deleted "create_bytes_to_underline_text" and "create_bytes_to_italicize_text" and made sure they are not being called anywhere.

Reply:
I tested the code with main.py and it works fine.